paddle/fluid/operators/controlflow/tensor_array_read_write_op.cc

```diff
@@ -140,10 +140,16 @@ class ReadFromArrayOp : public ArrayOp {
   void RunImpl(const framework::Scope &scope,
                const platform::Place &place) const override {
     auto *x = scope.FindVar(Input("X"));
-    PADDLE_ENFORCE(x != nullptr, "X must be set");
+    PADDLE_ENFORCE_NOT_NULL(
+        x,
+        platform::errors::InvalidArgument(
+            "X(Input Variable) must be set when we do read array operation"));
     auto &x_array = x->Get<framework::LoDTensorArray>();
     auto *out = scope.FindVar(Output("Out"));
-    PADDLE_ENFORCE(out != nullptr, "Out must be set");
+    PADDLE_ENFORCE_NOT_NULL(
+        out,
+        platform::errors::InvalidArgument(
+            "Out(Output Varibale) must be set when we do read read operation"));
```

Comment:
I understand the meaning of "do read array operation" but a better expression can be "when we call read operation" or "when we use read operation"

Reply:
Done Thanks